.gitignore

```diff
@@ -37,6 +37,8 @@ bld/
 # Uncomment if you have tasks that create the project's static files in wwwroot
 #wwwroot/
 
+.vscode/
```

Comment:
Recommend changing this to the standard block from <https://github.com/github/gitignore/blob/main/VisualStudio.gitignore>:

```
# VS Code files for those working on multiple tools
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
*.code-workspace
```

Otherwise we're going to start ignoring the files used for recommended extensions.

Reply:
@tomjschneider updated gitignore with this suggestion. Thanks for pointing this out!